Makefile

```diff
@@ -170,3 +170,8 @@ multimod-verify: $(MULTIMOD)
 multimod-prerelease: $(MULTIMOD)
 	multimod prerelease -s=true -b=false -v ./versions.yaml -m tools
 	$(MAKE) tidy
+
+.PHONY: add-tags
+add-tags: | $(MULTIMOD)
+	@[ "${MODSET}" ] || ( echo ">> env var MODSET is not set"; exit 1 )
```

Comment:
what's MODSET expected to be?

Reply:
Currently, it would only ever be `tools`:

https://github.com/open-telemetry/opentelemetry-go-build-tools/blob/1229bc4444fbfdb7160eee6d85042db2a37edcf9/versions.yaml#L16